functable.c

```diff
@@ -250,6 +252,7 @@ Z_INTERNAL uint32_t chunksize_stub(void) {
 Z_INTERNAL uint8_t* chunkcopy_stub(uint8_t *out, uint8_t const *from, unsigned len) {
     // Initialize default
     functable.chunkcopy = &chunkcopy_c;
+    cpu_check_features();
```

Comment:
Until we have benchmark for `chunkcopy`, this might still be an overkill.

Reply:
Possibly.  I was failing tests when I implemented one from memcpy so I think there's some strange side effects I don't seem to be reproducing that it relies on.

Reply:
Codecov is usually pretty accurate at finding dead code... Sometimes, it's a configuration issue; sometimes, it's just that the compiler knows it has no effect because the code block was already executed from another function or code path and it has no side-effects that change between runs...